NuCivic/dkan
# Harvest fixes CIVIC-4925 CIVIC-3838 CIVIC-4450

Issue: CIVIC-4925 CIVIC-3838 CIVIC-4450
## Description

**CIVIC-3838** Move harvest styles into theme - css should live in the theme unless absolutely required for functionality

**CIVIC-4450** The 'URL' field on the Source view page is editable - it can't be saved but it is weird that a user can type in the field

**CIVIC-4925** The source URL in the sidebar has disappeared and is replaced with "@source"

## QA Tests
1. Create a harvest source, add a filter and a default, harvest, then view the source node
2. Confirm that the source uri displays and is not editable
3. Confirm that the filter and default values are styled
4. Visit the source datasets page, confirm that the date input fields are 100px wide and display inline

<img width="972" alt="dkan" src="https://cloud.githubusercontent.com/assets/314172/20456249/6dd66730-ae38-11e6-946b-ac8e48724ed8.png">


1. Create a harvest source with errors
2. View the errors page, confirm that the ctools toggle bar works and is themed

<img width="1007" alt="errors___dkan" src="https://cloud.githubusercontent.com/assets/314172/20470140/3cfd1080-af6b-11e6-9181-4157424a7865.png">
